ServantPersistent/src/Domain/User.hs

```diff
@@ -17,9 +17,9 @@ import Operation
 createUser :: UserInput -> App (Either UserCreationError UserID)
 createUser u = runExceptT $ do
     time <- liftIO getCurrentTime
-    id <- ExceptT $ maybe (Left $ TenantDoesn'tExist (_tenantName u))
+    id <- ExceptT $ maybe (Left $ TenantDoesn'tExist (_tenantBackofficeDomain u))
                           Right
-                      <$> dbGetTenantByName (_tenantName u)
+                      <$> dbGetTenantByBackofficeDomain (_tenantBackofficeDomain u)
```

Comment:
this is fine for now. But, most probably, you will have to refactor this when you connect it to the JSON UI. The UI is anyways going to send you a tenant ID as part of the incoming JSON or as part of the URL.